app/utils/json_to_csv.js

```diff
@@ -81,6 +81,20 @@ module.exports = function (data) {
           value: row => {
             return dates.utcToTime(row.created_date)
           }
+        },
+        {
+          label: 'Corporate Card Surcharge',
+          value: row => {
+            const amountInPence = row.corporate_card_surcharge ? row.corporate_card_surcharge : 0
+            return (parseInt(amountInPence) / 100).toFixed(2)
+          }
+        },
+        {
+          label: 'Total Amount',
+          value: row => {
+            const amountInPence = row.total_amount ? row.total_amount : row.amount
+            return (row.transaction_type === 'refund') ? (parseInt(amountInPence) * -1 / 100).toFixed(2) : (parseInt(amountInPence) / 100).toFixed(2)
```

Comment:
We have helper functions for this in https://github.com/alphagov/pay-selfservice/blob/master/app/utils/currency_formatter.js for example https://github.com/alphagov/pay-selfservice/blob/e869a5511523b04ad476e482973d136ae657430e/app/models/TransactionEvent.class.js#L26

Reply:
Also just noticed, perhaps I wrong, but it says `amountInPence` but you dividing the pence amount by 100 so looks like it’s being transformed to pounds and pence, so the name is misleading